database_refresh/02-cronjob.yaml

```diff
@@ -10,6 +10,7 @@ spec:
   failedJobsHistoryLimit: 3
   jobTemplate:
```

Comment:
this batch is run every sunday. So 4 days to clean the job should still be viable.